docs/sources/flow/reference/components/discovery.kubernetes.md

```diff
@@ -388,3 +393,39 @@ values.
 ### Debug metrics
 
 `discovery.kubernetes` does not expose any component-specific debug metrics.
+
+## Examples
+
+### In-cluster discovery
+
+This example uses in-cluster authentication to discover all pods:
+
+```river
+discovery.kubernetes "k8s_pods" {
+  role = "pod"
+}
+```
+
+### Kubeconfig authentication
+
+This example uses a kubeconfig file to authenticate to the Kubernetes API:
+
+```river
+discovery.kubernetes "k8s_pods" {
+  role = "pod"
+  kubeconfig_file = "/path/to/kubeconfig"
```

Comment:
PATH_TO_KUBECONFIG or PATH?

Reply:
Ah, this is supposed to be an example so it's not a placeholder. I'll make up a fake path.